lua/toggleterm/terminal.lua

```diff
@@ -323,14 +323,21 @@ function Terminal:send(cmd, go_back)
   end
 end
 
-function Terminal:clear() self:send("clear") end
+-- Clear the terminal
+function Terminal:clear() 
+	if vim.fn.has('win32') == 1 then
```

Comment:
Three is already a variable in this file called `is_windows` that does this check so we don't need to do this logic again.

Also I think we could just change the command e.g.
```lua
local clear = is_windows and "cls" or "clear"
self:send(clear)
```